ANSWERS.md

```diff
@@ -1,6 +1,15 @@
 ### Questions:
 1. What is the DOM?
+Document Object Model that allows users to access and update information/styles using Javascript
+
 2. What is an event?
+What can be used to add functionality to webpages from the user clicking a webpage or hovering a mouse over some information
+
 3. What is an event listener?
+The function behind the event that allows the event to occur
+
 4. Why would we convert a NodeList into an Array?
-5. What is a component? 
+Nodelists tend to have as their first item a textnode which is not what a user would want when working with the DOM
+
+5. What is a component?
+Classes/Functions used to create code that can be reused to make items for the DOM and prevent DRY code
```

Comment:
Great job mentioning DRY principles. That is indeed a major reason for components.